services/task-svc/internal/room/models/models.go

```diff
@@ -23,4 +23,5 @@ type Bed struct {
 	Room           *Room           `gorm:"foreignKey:ID"`
 	Patient        *models.Patient `gorm:"foreignKey:BedID"`
 	OrganizationID uuid.UUID       `gorm:"column:organization_id"`
+	Name           string          `gorm:"column:name"`
 }
```

Comment:
the bed model should be in `bed/models/` and not `room/models`

Reply:
I replaced the bed/models Bed struct with an alias to this room/models Bed struct. There were two Bed structs before. It has to be here because it references Room, and structs in Room reference Bed, which would cause an import cycle, which is disallowed in go. In the future we should refactor this to a single models package.